_build/stac.hbs

```diff
@@ -35,6 +35,12 @@
       "href": "{{rootUrl}}stac/{{Slug}}.json",
       "rel": "self"
     },
+    {
+      "href": "{{rootUrl}}/{{Slug}}",

```

Comment:
Looking at how the URL in line 35 is composed we don't need the "/" here as the rootUrl will have it at the end already.

Reply:
It was removed.